_data/simple-icons.json

```diff
@@ -7049,6 +7049,11 @@
             "hex": "FAB040",
             "source": "https://github.com/pre-commit/pre-commit.com/blob/f263cdbcf46f97e1bd6229f2ab6d27bf8290ca88/logo.svg"
         },
+        {
+            "title": "Prefect",
+            "hex": "5772B0",
```

Comment:
I'm unfamiliar with https://icolorpalette.com/color/prefect so I can't say if we can consider this as an official source. How did you get to this URL? Was it linked by the brand? If so, from what page? If not, did you manage to find this color anywhere on the website?